bot/cogs/verification.py

```diff
@@ -155,6 +161,20 @@ def bot_check(ctx: Context) -> bool:
         else:
             return True
 
+    @tasks.loop(hours=1.0)
```

Comment:
It strikes me as unnecessarily often to check every hour if it's been a week yet. that means we're doing 167 of these checks that return None for every one that does something.

I'm thinking something like 12 or even 24 hours is plenty. Even if the message is skewed by 23 hours every couple of weeks, it's not at all critical that this happens the same time every week or that there's exactly one week between each of these.